tests/mineral_test.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
```

Comment:
Please include an Apache License header on all of your code. Thank you @heidiaclayton